src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -307,6 +299,7 @@ const ProgrammeScript = props => {
       // Video context probably needs more info like, which clip/track in the sequence?
       // investigate how to set currentTime in video context
```

Comment:
You can remove this comment too 👍 
```suggestion
      // investigate how to set currentTime in video context
```